samples/04_gis_analysts_data_scientists/determining_site_suitability_for_oil_palm_plantation.ipynb

```diff
@@ -0,0 +1,640 @@
+{
```

Comment:
<p>In order to carry out the suitability analysis, we need to consider some of the favorable environmental and crop criteria settings: </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>